apps/demo/config/blocks/FeatureList/index.tsx

```diff
@@ -21,7 +21,7 @@ const icons = Object.keys(reactFeather).reduce((acc, iconName) => {
   return acc;
 }, {});
 
-const iconOptions = Object.keys(reactFeather).map((iconName) => ({
+const iconOptions = Object.keys(reactLucide).map((iconName) => ({
```

Comment:
Hm, this does make for an impractically long list... I wonder if we should narrow it down. Can probably be done at a later date.

Reply:
> Hm, this does make for an impractically long list... I wonder if we should narrow it down. Can probably be done at a later date.

@chrisvxd Yeah, maybe for a single dropdown… Lucide is currently 1300+ icons vs Feather’s ~280! Although it looks from the [preview](https://puck-demo-git-fork-danielbayley-lucide-measured.vercel.app/edit) like there is somehow a lot of duplication going on with different names, so maybe it won’t be too bad with that fixed… Will have to dig into what is going wrong there…

Also since this is using Next, I should probably refactor to use `dynamicIconImports`, based on this: https://lucide.dev/guide/packages/lucide-react#with-dynamic-imports.

Reply:
Yeah looks good! Although I can't test it because the build is failing. Please see my other comment.

Reply:
Yeah it was kind of done quickly and flying blind… Haven't properly tested it myself yet! Will be away until Thu, so will fix then…

Reply:
Sometimes we must fly blind 🦅

No problem - just ping me when you're ready!

Reply:
> just ping me when you're ready!

@chrisvxd Should be good to go now! At least, I have it working locally…